cypress/integration/checkLists.ts

```diff
@@ -0,0 +1,31 @@
+describe('On check-lists example', () => {
+  it('checks the bullet when clicked', () => {
+    cy.visit('examples/check-lists')
+
+    cy.get('div[data-slate-node="element"]').should('have.length', 6)
```

Comment:
You re-use the string `'div[data-slate-node="element"]'` a few times, so put it in a variable with a decent name so it's clearer what's going on in the test
```suggestion
    const slateNodeElement = 'div[data-slate-node="element"]'

    cy.get(slateNodeElement).should('have.length', 6)
```